queue/src/fiskaltrust.Middleware.Localization.QueueDEFAULT/JournalProcessorDEFAULT.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using fiskaltrust.ifPOS.v1;
+using fiskaltrust.Middleware.Contracts;
+using Microsoft.Extensions.Logging;
+
+namespace fiskaltrust.Middleware.Localization.QueueDEFAULT
+{
+    public class JournalProcessorDEFAULT : IMarketSpecificJournalProcessor
+    {
+        private readonly ILogger<JournalProcessorDEFAULT> _logger;
+
+        public JournalProcessorDEFAULT(
+            ILogger<JournalProcessorDEFAULT> logger)
+        {
+            _logger = logger;
+        }
+
+        public IAsyncEnumerable<JournalResponse> ProcessAsync(JournalRequest request)
+        {
+            throw new NotImplementedException();
```

Comment:
What are we usually returning when a JournalType is not processable?

Reply:
In Italy we also just throw ATM.

I Germany we return this if the case does not match https://github.com/fiskaltrust/middleware/blob/main/queue/src/fiskaltrust.Middleware.Localization.QueueDE/JournalProcessorDE.cs#L104-L111.

I think returning something similar makes sense but I would only do that on case `0x...0000` and still trow (with a nice error log) if the case does not match.

Reply:
Maybe we can extract that to a `JournalCommand` like we do with the receipt commands 🤔